src/main/resources/Options/OptionsPlayer1.xml

```diff
@@ -1,9 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<options>
-	<volume>100</volume>
-	<controls>
-		<left>Left</left>
-		<right>Right</right>
-		<shoot>Up</shoot>
-	</controls>
-</options>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><options><volume>0.0</volume><controls><left>Left</left><right>Right</right><shoot>Up</shoot></controls></options>
```

Comment:
It's the layout from the the writer, I commited this by accident. The content is the same, only the layout is different.